lib/rouge/lexers/opentype_feature_file.rb

```diff
@@ -41,17 +47,18 @@ def self.keywords
           push :featurename
         end
         # } <tag> ;
-        rule %r/(\})((?:\s)*)/ do
+        rule %r/(\})((?:\s))/ do
           groups Punctuation, Text
           push :featurename
         end
         # solve include( ../path)
-        rule %r/(include)/i, Keyword, :includepath
+        rule %r/(include)(?!_dflt)/i, Keyword, :includepath
 
         rule %r/[\-\[\]\/(){},.:;=%*<>']/, Punctuation
 
         rule %r/`.*?/, Str::Backtick
         rule %r/\"/, Str, :dqs
+        rule %r/\\[^.*\s]+/i, Str::Escape
```

Comment:
Is this to match the `\736` in `VertOriginY \736 867;`? Is that really an escaped string?

Reply:
Not just to match things like `\736`.  In this case any name can be escaped like this. If it's a number like `\736` then it means that's the glyph ID, but it could also be `\somename` or it could be named the same as a keyword (very bad and not recommended of course, but possible) so it could be `\table` and the escape makes it clear it's a glyph name and not the keyword `table`. Some implementations escape every single name which is kind of ugly, but I suppose a safe way to avoid any conflicts with keywords.